### DISCUSSION TITLE
RST to hdf5

### DISCUSSION THREAD

**Discussion by Soyotec-Neo** [ansys/pydpf-core#1701] [Q&A]
How can I transfer all the information in the rst file to the hdf5 file, not just the stress and displacement？Refer to https://github.com/ansys/pydpf-core/blob/master/examples/05-file-IO/00-hdf5_double_float_comparison.py

**Comment by rlagha** [comment]
Hi @Soyotec-Neo , did you tried the "migrate_file" operator?
https://dpf.docs.pyansys.com/version/stable/operator_reference_load.html

@oparreno
